src/librustc_trans_traits/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
```

Comment:
Why does this exist as a separate crate from `rustc_trans_utils`? They are both depended upon by exactly `rustc_trans` and `rustc_driver`, and they semantically perform a very similar task.

Reply:
I was thinking about utils to be used by backends only, while traits contains the glue between backends and rustc_driver. I can merge them though.